app/design/frontend/Magento/blank/web/css/source/_navigation.less

```diff
@@ -8,6 +8,7 @@
 //  _____________________________________________
 
 @active-nav-indent: 54px;
+@_shadow: 0 0 5px 0 rgba(50, 50, 50, .75);
```

Comment:
I think now it could be overridden with any other file as all files will be merged together. Isn’t it?